src/components/utils/include/utils/log_message_loop_thread.h

```diff
@@ -35,6 +35,8 @@
 
 #include <string>
 #include <queue>
+#include <list>
+#include <deque>
```

Comment:
@LuxoftAKutsan Why did you added includes above? I can't see appropriate usage .